README.md

```diff
@@ -98,23 +100,60 @@ Exit Code Number   | Meaning
 `126`              | A service inside the box is not running / Command invoked cannot execute
 
 
-## IP Address Detection <a name="ip_addr"></a>
+<a name="ip-address-detection"></a>
+## IP address detection
 
 There is no standardized way of detecting Vagrant box IP addresses.
 This code uses the last IPv4 address available from the set of configured addresses that are *up*.  i.e. if eth0, eth1, and eth2 are all up and have IPv4 addresses, the address on eth2 is used.
 
 
-## Getting Involved with the Project <a name="Contributing"></a>
+<a name="development"></a>
+# Development
 
-We welcome your input. You can submit issues or pull requests with respect to the vagrant-service-manager plugin. Refer to the [contributing guidelines](https://github.com/projectatomic/vagrant-service-manager/blob/master/CONTRIBUTING.md) for detailed information on how to contribute to this plugin.
+<a name="setup"></a>
+## Setup
 
-You can contact us on:
-  * IRC: #atomic and #nulecule on freenode
-  * Mailing List: container-tools@redhat.com
+After cloing the repository, install the [Bundler](http://bundler.io/) gem:
 
+    $ gem install bundler
 
-## Builds <a name="builds"></a>
+Then setup your project dependencies:
 
-- Gem: https://rubygems.org/gems/vagrant-service-manager
+    $ bundle install
+
+The build is driven via rake. All build related tash should be executed in the
+Bundler environment, e.g. `bundle exec rake clean`. You can get a list of available
+Rake tasks via:
 
+    $ bundle exec rake -T
+
+<a name="acceptance-tests"></a>
+## Acceptance tests
+
+The source contains also a set of [Cucumber](https://cucumber.io/) acceptance tests. They can be run via:
+
+    $ bundle exec rake features
+
+The tests assume that the ADB and CDK box files are available under
+_build/boxes/adb-\<provider\>.box_ resp _build/boxes/cdk-\<provider\>.box_. You can
+either copy the box files manually or use the _get_adk_ resp. _get_cdk_ Rake tasks.
+
+After test execution the Cucumber test reports can be found under _build/features_report.html_
```

Comment:
Not quite sure how this enhances the user experience, but I'll add one. Note, there is more typing involved opening the browser via Rake instead of directly. 

In case you want to display everything in the console you can also execute `bundle exec rake features:pretty`